api/v1alpha6/conversion.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
I don't know the policy, but should we use the year of file creation?
```suggestion
Copyright 2022 The Kubernetes Authors.
```